### PR TITLE
Devenv: Add loki self logging to Jaeger block

### PR DESCRIPTION
**Pull request by aocenas**
Makes it easier to test going from logs to tracing by pushing Loki traces to Loki and also logging the trace id in Loki logs.

As the tracing feature is still behind feature flag I cannot add datasource config for it just yet as that would error out if you do not have the feature flag on.

### CODE REVIEW

File: devenv/docker/blocks/jaeger/docker-compose.yaml [hunk 1]
```diff
@@ -10,6 +10,11 @@
     ports:
       - "3100:3100"
     command: -config.file=/etc/loki/local-config.yaml
+    # For this to work you need to install the logging driver see https://github.com/grafana/loki/tree/master/cmd/docker-driver#plugin-installation
+    logging:
```
Comment (aocenas) [inline devenv/docker/blocks/jaeger/docker-compose.yaml:14]: @cyriltovena does this make sense? It works but maybe there is an easier way to do this.

Comment (cyriltovena) [reply devenv/docker/blocks/jaeger/docker-compose.yaml:14]: Alternatively you can add promtail or fluentbit to the stack, but this probably the easiest yes and fastest to setup, plus it adds nice labels for compose.

**Comment by aocenas** [comment]
This needs https://github.com/grafana/grafana/pull/23834 and https://github.com/grafana/grafana/pull/23829 first.

**Comment by s-h-a-d-o-w** [comment]
Just tried to run it, didn't see it in the data source select, so I went in the settings where I did see it but when I tried looking at the details, I got a:
`backend_srv.ts:142 Uncaught {message: "Plugin not found, no installed plugin with that id"}`

I did install that log driver plugin just as described in the docs though if that's what is meant here.

I also tried to shut down all sources and start only jaeger to doublecheck that there was no error during startup. Don't know what the "default driver" refers to here but anyway, that's my output:
```
Creating network "devenv_default" with the default driver
Creating devenv_loki_1     ... done
Creating devenv_jaeger_1   ... done
Creating devenv_promtail_1 ... done
```

**Review by davkal — COMMENTED** [review]
Loki logging to Loki works great, but I can't load the zipkin datasource (does this need rebasing?).

File: devenv/README.md [hunk 2]
```diff
@@ -40,3 +40,6 @@ make devenv sources=postgres,openldap,grafana postgres_version=9.2 grafana_versi
 ```
 
 Note: The grafana block is pre-configured with the dev-datasources and dashboards.
+
+When using jaeger block it will run both jaeger and loki that is sending traces to it so there is no need to run loki
+ separately. It will also log loki logs back to itself with docker logging driver which you need to install first see https://github.com/grafana/loki/tree/master/cmd/docker-driver#plugin-installation.
```
[the remark this answers is not in the document]
Comment (aocenas) [reply devenv/README.md]: Good point, changed.

**Comment by aocenas** [comment]
Yes this needed rebasing with master, sorry for that, should work fine now.

**Review by davkal — APPROVED** [review]
lgtm

**Pull request merged**